01_fib/jimbotech/ex_fib_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestFib(t *testing.T) {
+
+	fibResults := []int{1, 1, 2, 3, 5, 8, 13, 21, 34}
+
+	f := fibonacci()
+
+	for _, val := range fibResults {
+		if r := f(); r != val {
+			t.Errorf("Returned value %v does not match %v", r, val)
```

Comment:
error messages in Go should start with lower case letters. This is because error messages are often wrapped as the propagate causing a SeQuEnCe of lower/upper case letters: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Done